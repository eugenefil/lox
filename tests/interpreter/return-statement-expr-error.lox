fn f() {
    return x;
}
f();
