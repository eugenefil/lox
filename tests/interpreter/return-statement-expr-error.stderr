error: identifier 'x' is not defined
 --> $DIR/return-statement-expr-error.lox:2:12
  |
2 |     return x;
  |            ^
